pkg/collector/corechecks/servicediscovery/module/envs.go

```diff
@@ -30,6 +33,36 @@ const (
 	memFdMaxSize = 65536
 )
 
+const (
+	// maxSizeEnvsMap - maximum number of returned environment variables
+	maxSizeEnvsMap = 400
+	// defSizeEnvsMap - default size for environment variables map
+	defSizeEnvsMap = 200
+	// defSizeReadBuf - default buffer size for reading from proc/pid/environ
+	defSizeReadBuf = 2048
+	// defSizeEnvBuf - default buffer size for storing processed environment variable
+	defSizeEnvBuf = 100
+	// envVarDdService - name of target environment variable DD_SERVICE
+	envVarDdService = "DD_SERVICE"
+	// envVarDdTags - name of target environment variable DD_TAGS
+	envVarDdTags = "DD_TAGS"
+	// envVarDdInjectionEnabled - name of target environment variable DD_INJECTION_ENABLED
+	envVarDdInjectionEnabled = "DD_INJECTION_ENABLED"
+	// envVarDiscoveryEnabled - name of target environment variable DD_DISCOVERY_ENABLED
+	envVarDiscoveryEnabled = "DD_DISCOVERY_ENABLED"
+	// envVarOtelServiceName - name of target environment variable OTEL_SERVICE_NAME
+	envVarOtelServiceName = "DD_OTEL_SERVICE_NAME"
+)
+
+// map of environment variables of interest
+var targetEnvs = []string{
```

Comment:
This list should include all the environment variables needed for service name and instrumentation detection (which is why the tests are failing currently with this PR). There is an initial list in https://datadoghq.atlassian.net/browse/USMON-1126 but it should be double-checked.

Reply:
I've added some env. vars I discovered in the code and tests. However variables mentioned in the ticket USMON-1126 include variables which are not currently used. Please confirm if indeed need to include all variables in this ticket to env. extraction.

Reply:
I've added all env. variables I've found in the code and unit tests